### ISSUE TITLE
Estimate slippage with api

### ISSUE THREAD

**Issue by pguardiario** [synapsecns/synapse-api#56]
Is there a way to estimate slippage?

**Comment by bholagabbar** [comment]
> Is there a way to estimate slippage?

Unfortunately not at the moment I don't think. Can ping @bleaker0x and ask if the SDK has a method that supports this ?

**Comment by bleaker0x** [comment]
There isn't a function for this which is publicly exposed, but the functionality is already available. @metagabbar I'll work on this today and ping you. 

@pguardiario Just for clarification, are you looking to estimate slippage for same-chain token swaps, or cross-chain bridge swaps?

**Comment by bholagabbar** [comment]
> There isn't a function for this which is publicly exposed, but the functionality is already available. @metagabbar I'll work on this today and ping you.
> 
> @pguardiario Just for clarification, are you looking to estimate slippage for same-chain token swaps, or cross-chain bridge swaps?

I think cross chain bridge swaps would be most useful. When you say same chain token swaps I'm assuming the stable swap pool. Don't think that's as useful right now

**Comment by pguardiario** [comment]
Yes, cross chain swaps for different tokens or for the same token. I'm not
sure how the math on that works.
